CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
```

Comment:
Maybe you want to add a few entries for previous releases; at least the first release should be here.
What is OOM?

Reply:
order-of-magnitude. I'll convert this into a release entry when creating a release.